[TW-1220] Trigger token scan (#4921)
postmanlabs/postman-docs: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md

> 1. Select **Workspaces** from the header, then select **Create Workspace**.

Reviewer: ```suggestion
1. From the top menu, select **Workspaces > Create Workspace**.
```

    <img alt="Workspace menu" src="https://assets.postman.com/postman-docs/v10/workspace-switcher-v10.jpg" width="500px"/>

1. Specify a workspace **Name** and **Summary**.
    > Within a team, you can't have two public workspaces with the same name.
1. Under **Visibility**, select **Public**.

>     > If you are on a [Postman Professional or Enterprise plan](https://www.postman.com/pricing), you need approval from a user with the [Community Manager role](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) to create a public workspace. Add an optional note and select **Create Workspace** to send the request and create a team workspace until a Community Manager approves it. Sending the request also triggers the [Secret Scanner](/docs/administration/token-scanner/).

Reviewer: ```suggestion
    > If you are on a [Postman Professional or Enterprise plan](https://www.postman.com/pricing), creating a public workspace requires a [Community Manager](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) approval. Add an optional note and select **Create Workspace**. A request is sent to the Community Manager. Sending the request also triggers the [Secret Scanner](/docs/administration/token-scanner/).
```

Reviewer 2: @petewitcher-postman I'm not sure how GitHub handles a second suggestion on the same line. But I wanted to suggest that we change "you are" to "you're".

Also, I think you're already planning to do this, but I wanted to add here to reference later: I think we should either remove the explanation about selecting "Create Workspace", or update this to "select Create or Create Workspace". The reason is that users will see one of the CTAs depending on the plan type they're using. For more details, please see https://github.com/postmanlabs/postman-docs/pull/4915

Author: Agreed the creating/changing visibility procedures were incongruent so I just added a ## header to the changing visibility procedure. I think that addresses the issue, do you?

Reviewer 2: Looks good!

1. Select **Create workspace**.

    <img alt="Create public workspace" src="https://assets.postman.com/postman-docs/v10/create-public-wkspc-v10.jpg" width="500px"/>

> > You can use workspace templates to help set up your new public workspace. For more information, see [Creating workspaces with a template](/docs/collaborating-in-postman/using-workspaces/creating-workspaces/#creating-workspaces-with-a-template).

Reviewer: ```suggestion
> You can set up your new public workspace using a template. For more information, see [Creating workspaces with a template](/docs/collaborating-in-postman/using-workspaces/creating-workspaces/#creating-workspaces-with-a-template).
```

Author: Good suggestion, thank you. This content has been deleted.

> You must be a workspace [Admin](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles) to change the visibility of a workspace.
<!--  -->

> > If you are on a [Postman Professional or Enterprise plan](https://www.postman.com/pricing), you need approval from a user with the [Community Manager role](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) to change a workspace's visibility. In the **Overview** tab, select **Workspace Settings**. Under **Visibility**, select **Public** from the dropdown list, then **Request to Change Visibility**.

Reviewer: ```suggestion
> If you are on a [Postman Professional or Enterprise plan](https://www.postman.com/pricing), changing workspace visibility requires a [Community Manager](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) approval. To request this change, select **Workspace Settings**, then under **Visibility**, select **Public > Request to Change Visibility**.
```

Author: Good suggestion, thank you. This content has been rewritten as a procedure.
